vscode/package.json

```diff
@@ -1,19 +1,32 @@
 {
   "name": "dataform",
```

Comment:
nit: also seems to be mostly capitalized, so let's do that

Reply:
The name is required to be lowercased, we can add a displayName if we want